androidbrowserhelper/build.gradle

```diff
@@ -51,7 +51,7 @@ android {
 }
 
 dependencies {
-    api 'androidx.browser:browser:1.4.0'
+    api 'androidx.browser:browser:1.6.0-alpha02'
```

Comment:
We don't want to move all of `android-browser-helper` over to a non-stable version of `androidx.browser` - can you just make the demo depend on it (especially since the demo doesn't actually depend on the main `android-browser-helper` library at the moment.

Reply:
It uses LauncherActivity but we can replace it.